projects/go-lib/src/lib/go-config.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { GoConfigInterface } from './go-config.model';
+
+@Injectable()
+export class GoConfigService {
+  config: BehaviorSubject<GoConfigInterface> = new BehaviorSubject<GoConfigInterface> ({
+    brandColor: '#65B360'
+  });
+
+  public setBrandColor(color: string): void {
```

Comment:
I like this idea of having utility methods on the service to set new values, 🅰️ ➕